css/footer.css

```diff
@@ -0,0 +1,50 @@
+/* footer */
+footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-top: 1px solid #ccc;
+  width: 50%;
+  margin: 6% auto;
+}
```

Comment:
* Great work, you have factorised CSS files and improved code reuse! 
* Correct use of flex-box layout
* Could consider using relative units instead of fixed pixels to improve responsiveness
* Are you using Block Element Modifier (BEM) style CSS naming convention? 
* Use predefined CSS collar names and standardise use of (3) #hhh or (6) #hhhhhh collar hex code